templates/controllers/grid/settings/user/form/userDetailsForm.tpl

```diff
@@ -29,27 +29,38 @@
 	<input type="hidden" id="sitePrimaryLocale" name="sitePrimaryLocale" value="{$sitePrimaryLocale|escape}" />
 	<div id="userDetailsFormContainer">
 		<div id="userDetails" class="full left">
-			{if $userId}
-				<h3>{translate key="grid.user.userDetails"}</h3>
-			{else}
-				<h3>{translate key="grid.user.step1"}</h3>
+			{if !$userGroupUpdateOnly}
```

Comment:
Are you sure this change is correct? At a glance, I think the `{if}` check here was to determine whether we're creating a new user (`$userId` not present) vs. whether we're editing an existing one (`$userId` present).

Reply:
Oops, NVM, I see the nested `if`.